src/GitHub/Data/GitData.hs

```diff
@@ -188,7 +188,7 @@ data File = File
     , fileStatus    :: !Text
     , fileRawUrl    :: !(Maybe URL)
     , fileAdditions :: !Int
-    , fileSha       :: !Text
+    , fileSha       :: !(Maybe Text)
```

Comment:
related `<*> o .: "sha"` is better changed to `<*> o .:? "sha"`, except if there is really good reason to require `null` existence?

Reply:
Yes, good call! Fixed in ffed981c9d6ae616b7a2da5eb6a7f0523db9e1e1